cfpb/retirement
# Match the cfgov plurals config for translations

To fix a bug, Django 2.2.12 changed the way translation files are combined, and the
result changed how our retirement and cfgov Spanish translations worked together.

For the translations to work as before, both `django.po` files need to have the
same plural-translations scheme, which is defined at the top of the .po file.
cfgov had such a scheme defined (recently applied), and retirement did not,
putting them out of sync.

This adds a plural config line to retirement's django.po file so that the
definitions get properly merged.

## Testing
- Go to the Spanish wellbeing page on cfgov : <https://www.consumerfinance.gov/consumer-tools/financial-well-being/es/>
- Check answers for each question and hit the blue get-your-score button at bottom.
- Scroll to the bottom of results, and note that the first tab button, for Age, is blank – not translated.

<img width="345" alt="untranslated_button" src="https://user-images.githubusercontent.com/515885/91494877-98341300-e887-11ea-8e5c-f964c5dda8aa.png">


Install this branch in cfgov, do the same check locally at <http://localhost:8000/consumer-tools/financial-well-being/es/>

You should see the button translated as "Edad"

<img width="355" alt="translated_button" src="https://user-images.githubusercontent.com/515885/91494886-9d915d80-e887-11ea-9ffb-ea018f43c45d.png">
